tito.props.5.asciidoc

```diff
@@ -69,6 +69,12 @@ changelog_do_not_remove_cherrypick::
 If set to 0, it will not remove from cherry picked commits the part "(cherry
 picked from commit ...)"
 
+changelog_datetime::
```

Comment:
`changelog_date_with_timestamp` is probably a better name for this option...

Reply:
A timestamp is usually an integer (number of seconds from 1970). This name would be confusing because we don't want to put that timestamp into the changelog. Maybe rather `changelog_date_with_time`?

Reply:
Works for me!

Reply:
Updated